### DISCUSSION TITLE
Installing ubuntu rockchip on my CM3588

### DISCUSSION THREAD

**Discussion by tj-aiml** [Joshua-Riek/ubuntu-rockchip#981] [General]
Hello
A newbie here. Bought a CM3588 to build a NAS and not comfortable with their version of OS installation process due to security reasons. Upon researching I came across this community page. Can someone give me instructions on how do I install this ubuntu? I am also thinking of burning a usb but not sure how do I boot CM3588 from the usb? My goal is to install this ubuntu 22.04 OS on the eMMC. 

Please be as specific if you can.
Thanks in adv.

**Comment by Joshua-Riek** [comment]
CM3588 is not supported. 

Anyone could send a pull request to add support, I've reached out to the company for a dev kit and have not gotten a response, so not much I can do.

**Comment by nyanmisaka** [comment]
@tj-aiml [Someone in Armbian](https://github.com/armbian/linux-rockchip/pull/151) has already added cm3588 support, maybe you can contact him/her to take a look.
